services/web/server/src/simcore_service_webserver/users/_notifications.py

```diff
@@ -20,6 +20,7 @@ class NotificationCategory(StrAutoEnum):
     NEW_ORGANIZATION = auto()
     STUDY_SHARED = auto()
     TEMPLATE_SHARED = auto()
+    ANNOTATION_NOTE = auto()
```

Comment:
Please also add an entry in the `UserNotification` "examples" array with `ANNOTATION_NOTE` as category. These examples are used in tests.

Reply:
There is one in packages/models-library/src/models_library/projects_ui.py. I'll copy it around.